crates/astria-sequencer/src/app/tests_breaking_changes.rs

```diff
@@ -307,3 +352,178 @@ async fn app_execute_transaction_with_every_action_snapshot() {
 
     insta::assert_json_snapshot!(app.app_hash.as_bytes());
 }
+
+// Note: this test ensures that all actions are in their expected
+// bundleable vs non-bundleable category. Tests all actions except
+// IbcRelay.
+//
+// If any PR changes the bundleable status of an action, the PR must
+// be marked as breaking.
+#[allow(clippy::too_many_lines)]
+#[tokio::test]
+async fn app_transaction_bundle_categories() {
```

Comment:
I commented elsewhere, the tests here test implementation details of `astria-core` and are redundant.

It is implicitly understood that changes in behavior of `astria-core` are immediate breaking for `astria-sequencer` because they are a stepping stone.

Reply:
Ok, will remove.